lib/utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
yes, we have added this rule in the goodparts config:
https://github.com/dwyl/goodparts/issues/128
I'm not sure if we should use it, maybe we can reopen this issue on goodparts